contribs/gmf/less/layertree.less

```diff
@@ -69,15 +70,16 @@
     opacity: 0.5;
   }
 
-  .leaf {
+  .leaf, .group {
```

Comment:
Ok, I just took the example just above to write those lines. There is no line break. I'll put a line break.